src/main/java/hardcorequesting/team/PlayerEntry.java

```diff
@@ -118,7 +119,6 @@ public void write(JsonWriter out) throws IOException {
         out.endObject();
     }
 
-    @SideOnly(Side.SERVER)
```

Comment:
You can't just remove this, this line of code doesn't work client side (on MP when connected to a server)